supernotelib/cmds/supernote_tool.py

```diff
@@ -183,6 +184,7 @@ def main():
     parser_convert.add_argument('--exclude-background', action='store_true', default=False, help='exclude background and make it transparent (PNG and SVG are supported)')
     parser_convert.add_argument('--pdf-type', choices=['original', 'vector'], default='original', help='select PDF conversion type')
     parser_convert.add_argument('--no-link', action='store_true', default=False, help='disable links in PDF')
+    parser_convert.add_argument('--no-keyword', action='store_true', default=False, help='disable keywords in PDF')
```

Comment:
I'd like to set annotation feature disable by default
because annotation icons hide handwriting on Supernote tablets and make it hard to read.
This feature is useful on PC, but I would like to prioritize usability on Supernote tablets.

![20240804_205616](https://github.com/user-attachments/assets/df9d7df3-f013-4951-9f94-22ad52bf680b)